examples/Web-AC-control/Web-AC-control.ino

```diff
@@ -132,9 +132,19 @@ void handleFileUpload() {  // upload a new file to the FILESYSTEM
   if (upload.status == UPLOAD_FILE_START) {
     String filename = upload.filename;
     if (!filename.startsWith("/")) filename = "/" + filename;
+<<<<<<< HEAD
+    Serial.print("handleFileUpload Name: "); //Serial.println(filename);
+#if defined(ESP8266)
+    fsUploadFile = LittleFS.open(filename, "w");
+#else
+    fsUploadFile = SPIFFS.open(filename, "w");
+#endif
+    // Open the file for writing in SPIFFS (create if it doesn't exist)
+=======
```

Comment:
I think you've had a bad merge.

Reply:
Sorry I am not familiar with github merges, I thought it would only affect my own fork repository. Because this is just a test I submitted.

Reply:
Everything that you push to your master branch automatically ends up in this PR since you made the PR from master